[2025-01-25T09:38:01.307+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-25T09:38:01.322+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Email_POC.send_email manual__2025-01-25T09:37:52.993526+00:00 [queued]>
[2025-01-25T09:38:01.333+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Email_POC.send_email manual__2025-01-25T09:37:52.993526+00:00 [queued]>
[2025-01-25T09:38:01.333+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-25T09:38:01.371+0000] {taskinstance.py:2889} INFO - Executing <Task(EmailOperator): send_email> on 2025-01-25 09:37:52.993526+00:00
[2025-01-25T09:38:01.374+0000] {standard_task_runner.py:72} INFO - Started process 107 to run task
[2025-01-25T09:38:01.378+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'Email_POC', 'send_email', 'manual__2025-01-25T09:37:52.993526+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/send_email.py', '--cfg-path', '/tmp/tmphkrdx7mw']
[2025-01-25T09:38:01.380+0000] {standard_task_runner.py:105} INFO - Job 34: Subtask send_email
[2025-01-25T09:38:01.469+0000] {task_command.py:467} INFO - Running <TaskInstance: Email_POC.send_email manual__2025-01-25T09:37:52.993526+00:00 [running]> on host 41e0ee0a36b9
[2025-01-25T09:38:01.594+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Email_POC' AIRFLOW_CTX_TASK_ID='send_email' AIRFLOW_CTX_EXECUTION_DATE='2025-01-25T09:37:52.993526+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-25T09:37:52.993526+00:00'
[2025-01-25T09:38:01.595+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-25T09:38:01.643+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/utils/email.py:155 RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
[2025-01-25T09:38:01.644+0000] {email.py:272} INFO - Email alerting: attempt 1
[2025-01-25T09:38:03.456+0000] {email.py:283} INFO - Sent an alert email to ['adarshchintada2@gmail.com']
[2025-01-25T09:38:05.656+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-25T09:38:05.657+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Email_POC, task_id=send_email, run_id=manual__2025-01-25T09:37:52.993526+00:00, execution_date=20250125T093752, start_date=20250125T093801, end_date=20250125T093805
[2025-01-25T09:38:05.705+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-25T09:38:05.734+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-25T09:38:05.736+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
